src/stub/language/variable_name_options.rs

```diff
@@ -20,6 +20,9 @@ pub struct VariableNameOptions {
     #[serde(default)]
     allow_uppercase_vars: bool,
     keywords: Vec<String>,
+    /// Whether a keyword should escape case insensitive variations.
+    /// Defaults to case sensitive.
+    case_insensitive_keywords: Option<bool>,
```

Comment:
I would just make this `bool` with `#[serde(default)]`, similar to `allow_uppercase_vars` above

Reply:
Changed.